addons/web/static/src/less/kanban_view_mobile.less

```diff
@@ -29,7 +29,7 @@
             }
         }
 
-        &.o_kanban_small_column .o_kanban_group:not(.o_column_folded) {
+        &[class] .o_kanban_group:not(.o_column_folded) {
```

Comment:
ok but could you add a comment here explaining why you need to specify &[class]?

Reply:
done